src/lib.rs

```diff
@@ -1,11 +1,279 @@
+/*!
+This crate provides convenience methods for encoding and decoding numbers in
```

Comment:
Something is wrong with this doc comment. Can you copy the one from the original `src/lib.rs`?